miscellaneous/eggnog_performance.md

```diff
@@ -15,4 +15,34 @@ The results are summarized in the table below:
 | 5               | 1                             | N.A. (43 finished)        | 21 min                       |
 | 10              | 1                             | N.A. (39 finished)        | 42 min                       |
 
-Each node has 3 hrs of wall time.
\ No newline at end of file
+(Each node has 3 hrs of wall time.)
+
+### Test with scratch drive (Recommended by NERSC)
+
+Copied the source files to `$SCRATCH` and ran the same tests. The results are summarized in the table below:
+
+| Number of nodes | Parallelization within a node | Time to finish 50 genomes                         | Average node time per genome |
+|-----------------|-------------------------------|---------------------------------------------------|------------------------------|
+| 1               | 1                             | 54.83 mins                                        | 1.1 min                      |
+| 1               | 2                             | 39.77 mins (both batches)                         | 0.79 min                     |
+| 1               | 5                             | 27.92 mins (first batch), 37.11 mins (last batch) | 0.74 min                     |
+| 1               | 10                            | 22.62 mins (first batch), 34.7 mins (last batch)  | 0.69 min                     |
+| 1               | 15                            | 18.8 mins (first batch), 30.79 mins (last batch)  | 0.62 min                     |
+| 2               | 1                             | 25.91 mins (first node), 26.85 mins (last node)   | 1.06 min                     |
+| 5               | 1                             | 10.03 mins (first node), 13.24 mins (last node)   | 1.21 min                     |
```

Comment:
Seems like the next test is nodes > 1 + parallelization?

Reply:
hmmm. maybe. But I think that's not necessary. 1 parallelization with 1, 2, 5 nodes shows nodes are pretty identical as NERSC advised to us.

Reply:
If we really wanted to get crazy we could try and balance the batches via file size or protein count

Reply:
I wouldn't go that far. I am not even sure if anyone is looking at test results for the first 500 genomes. Seems like 10 or 15 threads per node is reasonable.

Reply:
so what's the next step(s)?

Reply:
I am planning to discuss the test result with the team first. And depends on the outcomes, plan next.